mods/cnc/maps/nod04b/nod04b.lua

```diff
@@ -32,7 +32,7 @@ Atk4Waypoints = { waypoint0, waypoint9 }
 Atk6ActorTriggerActivator = { Civilian1, Civilian2 }
 Atk5ActorTriggerActivator = { Soldier1, Soldier2, Soldier3, Actor105 }
 WinActorTriggerActivator = { GDICiv1, GDICiv2, GDICiv3, GDICiv4, GDICiv5, GDICiv6, GDICiv7, GDICiv8, GDICiv9, GDICiv11, GDICiv12, GDICiv13 }
-GcivActors = { Gcvi1, Gciv2, GDICiv2, GDICiv9, GDICiv10, GDICiv11 }
```

Comment:
Why remove this instead of adjusting to `Gciv1`?

Reply:
Well spotted.